probes/sastToolSonarInstalled/def.yml

```diff
@@ -0,0 +1,29 @@
+# Copyright 2023 OpenSSF Scorecard Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+id: sastToolSonarInstalled
+short: Check that the project uses Sonar.
+motivation: >
+  SAST is testing run on source code before the application is run. Using SAST tools can prevent known classes of bugs from being inadvertently introduced in the codebase.
+implementation: >
+  The implementation checks the projects pom.xml for use of Sonar.
+outcome:
+  - If the project uses Sonar, the probe returns one finding with OutcomePositive (1).
+  - If the project does not use the Sonar, the probe returns one finding with OutcomeNegative (0).
```

Comment:
nit: `use the Sonar` -> `use Sonar` to be consistent with previous line

Reply:
Corrected in https://github.com/ossf/scorecard/pull/3571/commits/120c5c60da09272b4bb8fc322029c01bc8756b26